ddtrace/tracer/stats.go

```diff
@@ -52,7 +52,7 @@ type concentrator struct {
 	buckets map[int64]*rawBucket
 
 	// stopped reports whether the concentrator is stopped (when non-zero)
-	stopped uint64
+	stopped atomic.Uint64
```

Comment:
This can be 32 bits. It could also be `atomic.Bool`. That type is internally an `atomic.Uint32`. However, that's technically not documented so maybe you don't want to depend on that?